core/templates/components/common-layout-directives/navigation-bars/top-navigation-bar.directive.html

```diff
@@ -18,71 +18,77 @@
+            <li class="nav-item"> <a class="dropdown-item nav-link oppia-navbar-tab-content protractor-test-get-started-link " href="/get_started" translate="I18N_TOPNAV_GET_STARTED"></a></li>
+            <li class="nav-item"> <a class="dropdown-item nav-link oppia-navbar-tab-content protractor-test-playbook-link " href="/teach#playbook" translate="I18N_TOPNAV_PARTICIPATION_PLAYBOOK"></a></li>
+            <li class="nav-item"> <a class="dropdown-item nav-link oppia-navbar-tab-content" href="https://medium.com/oppia-org" target="_blank" rel="noopener" translate="I18N_TOPNAV_BLOG">Blog</a></li>
+            <li class="nav-item"> <a class="dropdown-item nav-link oppia-navbar-tab-content"  href="/contact" translate="I18N_TOPNAV_CONTACT_US"></a></li>
+            <li class="nav-item"> <a class="dropdown-item nav-link oppia-navbar-tab-content" href="http://oppiafoundation.org"  target="_blank" rel="noopener" translate="I18N_TOPNAV_OPPIA_FOUNDATION"></a></li>
+            <li class="nav-item"> <a ng-keydown="$ctrl.onMenuKeypress($event, 'aboutMenu', {tab: $ctrl.ACTION_CLOSE})" class="dropdown-item nav-link oppia-navbar-tab-content" href="https://oppiafoundation.org/volunteer" target="_blank" rel="noopener" translate="I18N_TOPNAV_GET_INVOLVED"></a></li>
+          </ul>
+        </li>
+        <li ng-if="$ctrl.CLASSROOM_PAGE_IS_SHOWN"
+            ng-class="{'open' : $ctrl.activeMenuName === 'classroomMenu'}"
+            uib-dropdown
+            ng-show="$ctrl.navElementsVisibilityStatus.I18N_TOPNAV_CLASSROOM"
+            aria-hidden="false"
+            class="nav-item  oppia-navbar-clickable-dropdown protractor-test-classroom-oppia-list-item">
+          <a ng-keydown="$ctrl.onMenuKeypress($event, 'classroomMenu', {shiftTab: $ctrl.ACTION_CLOSE, enter: $ctrl.ACTION_OPEN})" class="nav-link oppia-navbar-tab dropdown-toggle"
+             role="menuitem"
+             aria-haspopup="true"
+             aria-expanded="false"
+             aria-hidden="false"
+             tabindex="0"
+             ng-mouseover="$ctrl.openSubmenu($event, 'classroomMenu')"
+             ng-mouseleave="$ctrl.closeSubmenuIfNotMobile($event)">
+            <span class="oppia-navbar-tab-content" translate="I18N_TOPNAV_CLASSROOM"></span>
+          </a>
+          <ul uib-dropdown-menu class="uib-dropdown-menu oppia-navbar-dropdown"
+              ng-mouseover="$ctrl.openSubmenu($event, 'classroomMenu')"
+              ng-mouseleave="$ctrl.closeSubmenuIfNotMobile($event)"
+              role="menu">
+            <li class="nav-item"> <a ng-keydown="$ctrl.onMenuKeypress($event, 'classroomMenu', {shiftTab: $ctrl.ACTION_CLOSE})" class="dropdown-item nav-link oppia-navbar-tab-content protractor-test-classroom-link" href="/math" translate="I18N_TOPNAV_CLASSROOM_BASIC_MATHS"></a></li>
```

Comment:
This line is really long: can you place the `a` tag on a new line and maybe put each of the attributes for the `a` tag on a separate line as well.

Reply:
Done